lib/Elevate/Leapp.pm

```diff
@@ -38,7 +38,7 @@ sub _build_yum ($self) {
     return Elevate::YUM->new( cpev => $self->cpev() );
 }
 
-sub install ($self) {
+sub install ( $self, $remove_kernel_devel = 1 ) {
```

Comment:
This seems like a red flag. Why are we doing component work here. Could it be easily moved?

Reply:
Yeah, we should move the kernel-devel uninstall to the code that calls install() when running the leapp upgrade.